src/services/rpc-service.ts

```diff
@@ -1,10 +1,10 @@
-import { ValidBlockData } from "../../types/handler";
+import { ChainId, ValidBlockData } from "../../types/handler";
```

Comment:
We generally refer to these as network id in our code because rollups aren't "chains."